ods_ci/tests/Tests/0600__distributed_workloads/0602__training/test-run-training-stack-tests.robot

```diff
@@ -8,26 +8,84 @@ Resource          ../../../../tasks/Resources/RHODS_OLM/install/oc_install.robot
 Resource          ../../../../tests/Resources/Page/DistributedWorkloads/DistributedWorkloads.resource
 
 
+
 *** Test Cases ***
-Run Training operator KFTO test with NVIDIA CUDA image
-    [Documentation]    Run Go KFTO test for Training operator using PyTorch job with NVIDIA CUDA image
+
+##Note: The testcases added below tagged with 'Kfto-MultiGpu' are meant for manual execution and must be skipped during QG tests
+
+Run Training operator KFTO Huggingface Trainer test with NVIDIA CUDA image using Single Node Single Gpu
+    [Documentation]    Run Go KFTO test for Training operator using PyTorch job with NVIDIA CUDA image using Single Node Single Gpu configuration
     [Tags]  Resources-GPU    NVIDIA-GPUs
-    ...     RHOAIENG-16035
     ...     Tier1
     ...     DistributedWorkloads
     ...     Training
     ...     TrainingOperator
-    Run Training Operator KFTO Test    TestPyTorchJobWithCuda    ${CUDA_TRAINING_IMAGE}
+    Run Training Operator KFTO Test    TestPyTorchJobSingleNodeSingleGpuWithCuda    ${CUDA_TRAINING_IMAGE}
 
-Run Training operator KFTO test with AMD ROCm image
-    [Documentation]    Run Go KFTO test for Training operator using PyTorch job with AMD ROCm image
+Run Training operator KFTO Huggingface Trainer test with NVIDIA CUDA image using Single Node Multi Gpu
+    [Documentation]    Run Go KFTO test for Training operator using PyTorch job with NVIDIA CUDA image using Single Node Multi Gpu configuration
+    [Tags]  Resources-GPU    NVIDIA-GPUs
```

Comment:
Remove these tags `Resources-GPU` and  `NVIDIA-GPUs` as this tag is used to executes all tests which requires gpus in Quality gates